test/oli_web/live/products_test.exs

```diff
@@ -93,4 +94,92 @@ defmodule OliWeb.ProductsLiveTest do
       assert has_element?(view, "a[href=\"#{Routes.discount_path(OliWeb.Endpoint, :product, product.slug)}\"]")
     end
   end
+
+  describe "product overview image upload" do
+    setup [:admin_conn, :create_product]
+
+    test "displays the current image", %{conn: conn, product: product} do
+      Sections.update_section(product, %{cover_image: "https://example.com/some-image-url.png"})
```

Comment:
NIT: For this cases, I prefer to not use the `product` that comes in the context and instead insert a new one with the data we need (so we not call app logic here).

Reply:
Agree, did think about it when doing tests for `page_delivery_controller` but they had a lot of logic in the creation of the section, besides just inserting into the table. In this case it makes sense because we can replicate the setup easily.